cmd/kyma/alpha/add/module/module.go

```diff
@@ -31,8 +31,8 @@ func NewCmd(o *Options) *cobra.Command {
 
 	cmd := &cobra.Command{
 		Use:   "module [name] [flags]",
-		Short: "Enables a module in the cluster or in the given Kyma resource.",
-		Long: `Use this command to enable Kyma modules available in the cluster.
+		Short: "Activates a module in the cluster or in the given Kyma resource.",
+		Long: `Use this command to activate Kyma modules available in the cluster.
```

Comment:
I would stick to `add`

```suggestion
		Short: "Add a module in the cluster or in the given Kyma resource.",
		Long: `Use this command to add Kyma modules available in the cluster.
```